### PR TITLE
[TT-1016] Swtich from gotestfmt to gotestloghelper

### PR DESCRIPTION
**Pull request by tateexon**
Makes go test log output less noisy when used. Cleans up things like testing.T.Log prefix, only shows errors, highlights errors in red, copies error messages to the top if there are lots of logs before the error to save you time from having to scroll to the bottom, adds drop downs in CI for individual tests so you can look at specific logs more easily without having to scroll through others. An example of some of the logs on errors: https://github.com/smartcontractkit/chainlink/actions/runs/8605875051/job/23583096855?pr=12633

### CODE REVIEW

File: .github/workflows/integration-tests.yml [hunk 1]
```diff
@@ -461,14 +463,16 @@ jobs:
       ## Run this step when changes that require tests to be run are made
       - name: Run Tests
         if: needs.changes.outputs.src == 'true'
-        uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/run-tests@7882cf348cd6a1f6bcf1ee8280185584ebba96e9 # v2.3.10
+        uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/run-tests@5dd916d08c03cb5f9a97304f4f174820421bb946 # v2.3.11
         with:
-          test_command_to_run: cd ./integration-tests && go test -timeout 30m -count=1 -json -test.parallel=${{ matrix.product.nodes }} ${{ steps.build-go-test-command.outputs.run_command }} 2>&1 | tee /tmp/gotest.log | gotestfmt
+          test_command_to_run: cd ./integration-tests && go test -timeout 30m -count=1 -json -test.parallel=${{ matrix.product.nodes }} ${{ steps.build-go-test-command.outputs.run_command }} 2>&1 | tee /tmp/gotest.log | gotestloghelper -ci -singlepackage
```
Comment (AnieeG) [inline .github/workflows/integration-tests.yml:468]: how about merging` tee /tmp/gotest.log | gotestloghelper -ci -singlepackage` to `test_command_to_run` if opted for `gotestloghelper` in run-test? Might help in maintenance and updating all run-tests. Can take an input to denote if gotesthelper is preferred.

File: .github/workflows/integration-tests.yml [hunk 2]
```diff
@@ -842,13 +848,15 @@ jobs:
...
         with:
-          test_command_to_run: cd ./integration-tests && go test -timeout 30m -count=1 -json ./migration 2>&1 | tee /tmp/gotest.log | gotestfmt
+          test_command_to_run: cd ./integration-tests && go test -timeout 30m -count=1 -json ./migration 2>&1 | tee /tmp/gotest.log | gotestloghelper -ci -singlepackage
           test_download_vendor_packages_command: cd ./integration-tests && go mod download
           cl_repo: ${{ env.CHAINLINK_IMAGE }}
           cl_image_tag: ${{ steps.get_latest_version.outputs.latest_version }}
-          artifacts_location: ./integration-tests/migration/logs
+          artifacts_location: |
+            ./integration-tests/migration/logs
+            /tmp/gotest.log
```
Comment (AnieeG) [inline .github/workflows/integration-tests.yml:859]: similar as before if gotesthelper is opted` /tmp/gotest.log` can be added to `artifacts_location`

**Review by AnieeG — APPROVED** [review]
approved with some nits

File: .github/workflows/integration-tests.yml (hunk 1 above)
Comment (tateexon) [reply .github/workflows/integration-tests.yml:468]: That is a good idea. I will be ripping out the run-tests from chainlink-github-actions and moving it to .github soon so I will add this to the changes that would improve the flow there!

**Review by AnieeG — APPROVED** [review]

**Pull request merged**